src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.cpp

```diff
@@ -73,8 +73,8 @@ void TargetedMovementGeneratorMedium<T, D>::_setTargetLocation(T* owner, bool up
             //   doing a "dance" while fighting
             if (owner->IsPet() && i_target->GetTypeId() == TYPEID_PLAYER)
             {
-                dist = i_target->GetCombatReach();
-                size = i_target->GetCombatReach() - i_target->GetObjectSize();
+                dist = 1.0f; //i_target->GetCombatReach();
```

Comment:
> These values are for the pet, not the player. All pets should have the same reach.

But in that case, i_target is the player, so the player's combat reach/object size are used.

It makes sense that pets stay further away from, say, a Tauren compared to a Blood Elf.